platform_global_teacher_campus/models.py

```diff
@@ -1,3 +1,33 @@
 """
 Database models for platform_global_teacher_campus.
 """
+
+from django.db import models
+from edxapp_wrapper.courses import get_course_overview
+
+CourseOverview=get_course_overview()
+class CourseCategory(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=100)
+    parentCategory = models.IntegerField()
+
+    def __str__(self):
+        return self.name
+
+class ValidationProcess(models.Model):
+    id = models.AutoField(primary_key=True)
+    courseId = models.ForeignKey(CourseOverview, on_delete=models.CASCADE)
+    categories = models.ManyToManyField(CourseCategory)
+    organizationId = models.IntegerField()
+    currentValidationUserId = models.IntegerField()
+    validationBodyId = models.IntegerField()
+
+    def __str__(self):
+        return f"Validation Process for Course {self.courseid}"
+
+class ValidationBody(models.Model):
+    id = models.AutoField(primary_key=True)
+    validator = models.CharField(max_length=50)
+    name = models.CharField(max_length=100)
+    admin_notes = models.CharField(max_length=100)
+    organizations = models.CharField(max_length=50)
```

Comment:
Why CharField?? this should be a relation of many to many with organizations model.